recipes-nemomobile/usb-moded/usb-moded_git.bb

```diff
@@ -28,7 +28,7 @@ RDEPENDS_${PN} += "buteo-mtp"
 do_configure_prepend() {
     sed -i "s@systemd-daemon@systemd@" configure.ac
     sed -i "s@shell@ceres@g" systemd/adbd-functionfs.sh
-    sed -i "s@ adbd.service@ android-tools-adbd.service@" ${S}/systemd/adbd-prepare.service
```

Comment:
The reason why I usually add a space before that sort of regexp is because if do_configure is run twice, the text would become android-tools-android-tools-adbd.service. (this can happen in some conditions)
To be fair, using sed in those prepare scripts has always been super dirty. But I won't blame you for prior bad design and I'll just merge your PR. :) Ideally, this should have been done with a patch instead.